enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -496,6 +644,55 @@ the updates, and continue enforcement rollout only when the policy on previous c
   }
   ```
 
+#### Development Phase 4
+
+- **Summary**
+
+  - Implement rollouts for policy sets.
+
+- **Details**
+
+  - Add an `OrderPolicies` boolean and the `RolloutStrategy` struct to the spec of the `PolicySet`.
+  - `OrderPolicies` signals to the `governance-policy-propagator` controller whether to inject a dependency on the
```

Comment:
Could an injected dependency between policies conflict with an actual dependency?

Reply:
The existing dependency would be added to the list of dependencies on the replicated policy (rather than a conflict per se), so both would block the policy until they're fulfilled. The dependencies are one point of complexity here, but for this it'd become a union of the dependencies.